tests/testthat/test-variation-across-groups.R

```diff
@@ -0,0 +1,356 @@
+context("Checking variationAcrossGroups")
```

Comment:
When pvalue and group (a,b,c) is output, you should test that it's calculated correctly.